homeassistant/components/geocaching/device_tracker.py

```diff
@@ -0,0 +1,51 @@
+"""Device tracker entities for Geocaching."""
+
+from typing import Any
+
+from geocachingapi.models import GeocachingCache
+
+from homeassistant.components.device_tracker.config_entry import TrackerEntity
+
+from .coordinator import GeocachingDataUpdateCoordinator
+from .entity import GeoEntityBaseCache
+
+
+# A tracker entity that allows us to show caches on a map.
+class GeoEntityCacheLocation(GeoEntityBaseCache, TrackerEntity):
+    """Entity for a cache GPS location."""
```

Comment:
In order for this to be a proper platform, this needs its own `async_setup_entry`. I would suggest to make these PRs even smaller